src/main/resources/schemas/workload.json

```diff
@@ -69,6 +69,16 @@
                             "type": "integer",
```

Comment:
These hopefully do not need to be included here so anyone can include any custom properties and run validation (this is related to the comment above that I left for `CustomTaskExecutorArguments` interface).